moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -246,7 +255,7 @@ class Robot
    */
   bool jointInterpolation(const std::unique_ptr<TrajectoryPoint> & traj_point,
            std::vector<trajectory_msgs::JointTrajectoryPoint> & points,
-           const unsigned int num_steps);
+           const unsigned int num_steps, bool collision_check = false);
```

Comment:
It is only checked [here once](https://github.com/shaun-edwards/moveit_simple/pull/22/files#diff-8ae580aedd7e6266898c159ae2e4a380R503).